sql-plugin/src/test/scala/com/nvidia/spark/rapids/HostAllocSuite.scala

```diff
@@ -351,8 +349,9 @@ class HostAllocSuite extends AnyFunSuite with BeforeAndAfterEach with
       // put RMM back for other tests to use
       Rmm.initialize(RmmAllocationMode.CUDA_DEFAULT, null, 512 * 1024 * 1024)
     }
-    // 1 GiB
-    PinnedMemoryPool.initialize(1 * 1024 * 1024 * 1024)
+    // less than 1 GiB, see more background at:
+    // https://github.com/NVIDIA/spark-rapids/issues/12194#issuecomment-2703186601
+    PinnedMemoryPool.initialize(500 * 1024 * 1024)
```

Comment:
In the issue it was suggested to use 900m as a workaround but here we are only allocating 500m. Was it not possible to even allocate 900m?

Reply:
Yes, I'm afraid so. The simple code snippet mentioned in the issue was run on the machine behind rapids_it-WITHOUT-isolation. But the machine behind pre-merge-test is a different one. I observed allocation failure when running premerge test for this PR, so I'm trying to see if reducing to 500M will help